plugins/hydra_nevergrad_sweeper/example/dummy_training.py

```diff
@@ -0,0 +1,33 @@
+# Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved
+import logging
+from typing import Any
+
+import hydra
+
+log = logging.getLogger(__name__)
+
+
+@hydra.main(config_path="config.yaml")
+def dummy_training(cfg: Any) -> float:
+    """A dummy function to minimize
+    Minimum is 0.0 at:
+    lr = 0.12, dropout=0.33, db=mnist, batch_size=4
+    """
+    do = cfg.dropout
+    bs = cfg.batch_size
+    out = float(
+        abs(do - 0.33) + int(cfg.db == "mnist") + abs(cfg.lr - 0.12) + abs(bs - 4)
+    )
+    logging.info(
+        "dummy_training(dropout=%s, lr=%s, db=%s, batch_size=%s) = %s",
+        do,
+        cfg.lr,
+        cfg.db,
+        bs,
+        out,
+    )
```

Comment:
1. will do
2. me too, but pylint screams at me (in my editor) when I do this: the reason is that logging does not interpolate if it does not need to (aka if the logs are not activated), so using fstring inside logging is considered "inefficient". If you still prefer fstrings, I'm fine with it though :D

Reply:
Yeah. it's a valid point but we are no where near code where this would make even a slight difference here :)

Let's stick to fstrings unless we have a good reason not to.